### PR TITLE
fix(menu-item): make the slot available to all roles (VIV-1236)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/menu-item/README.md [hunk 1]
```diff
@@ -148,13 +148,22 @@ The expanded state of the element
...
+When icon member is set, or role other than `role='menuitem'` - the icon or checkbox/radiobutton will be trailing.
 ```html preview
+<style>
+  html { /* for demo purposes */
+    block-size: 170px;
+  }
+</style>
 <vwc-menu open>
- <vwc-menu-item text="Export to PDF">
+ <vwc-menu-item text="avalable" role="menuitemradio">
```
Comment (rinaok) [inline libs/components/src/lib/menu-item/README.md:160]:
```suggestion
 <vwc-menu-item text="available" role="menuitemradio">
```

File: libs/components/src/lib/menu-item/README.md [hunk 2]
```diff
@@ -148,13 +148,22 @@ The expanded state of the element
 ## Slots
 ### meta
 Assign nodes to the `meta` slot to set a badge or an additional icon.  
-When icon member is set with slot - the icon will be trailing.
+Slot with icon or role other than `role='menuitem'` will make the
```
Comment (rinaok) [inline libs/components/src/lib/menu-item/README.md:151]:
```suggestion
Slot with icon or role other than `role='menuitem'` will make the 
```

will make what?

File: libs/components/src/lib/menu-item/ui.test.ts [hunk 3]
```diff
@@ -51,6 +51,12 @@ test('should show the component', async ({ page }: { page: Page }) => {
 				<vwc-menu-item icon="file-pdf-line" text="Export to PDF">
 				<vwc-badge slot="meta" appearance='subtle' connotation='cta' shape='pill' icon='check-solid'></vwc-badge>
 				</vwc-menu-item>
+				<vwc-menu-item text="avalable" role="menuitemradio">
```
Comment (rinaok) [inline libs/components/src/lib/menu-item/ui.test.ts:54]:
```suggestion
				<vwc-menu-item text="available" role="menuitemradio">
```

File: libs/components/src/lib/menu-item/ui.test.ts [hunk 4]
```diff
@@ -51,6 +51,12 @@ test('should show the component', async ({ page }: { page: Page }) => {
...
+				  <vwc-badge slot="meta" appearance='filled' connotation='success' shape='pill' icon='check-solid'></vwc-badge>
+				</vwc-menu-item>
+				<vwc-menu-item text="Bussy" role="menuitemcheckbox" checked>
```
Comment (rinaok) [inline libs/components/src/lib/menu-item/ui.test.ts:57]:
```suggestion
				<vwc-menu-item text="bussy" role="menuitemcheckbox" checked>
```

File: libs/components/src/lib/menu-item/README.md [hunk 5]
```diff
@@ -148,13 +148,22 @@ The expanded state of the element
...
+    block-size: 170px;
+  }
+</style>
 <vwc-menu open>
- <vwc-menu-item text="Export to PDF">
+ <vwc-menu-item text="avalable" role="menuitemradio">
   <vwc-badge slot="meta" appearance='filled' connotation='success' shape='pill' icon='check-solid'></vwc-badge>
   </vwc-menu-item>
-  <vwc-menu-item icon="check-line" text="Export to PDF">
+   <vwc-menu-item text="Away" role="menuitemradio">
```
Comment (rinaok) [inline libs/components/src/lib/menu-item/README.md:163]:
```suggestion
<vwc-menu-item text="Away" role="menuitemradio">
```

File: libs/components/src/lib/menu-item/README.md (hunk 2 above)
Comment (rachelbt) [reply libs/components/src/lib/menu-item/README.md:151]: LOL!
I'll fix it

**Review by rinaok — APPROVED** [review]

**Pull request merged**